aws-ts-pulumi-webhooks/index.ts

```diff
@@ -53,19 +55,23 @@ webhookHandler.get("/", async (_, res) => {
 });
 
 webhookHandler.post("/", logRequest, authenticateRequest, async (req, res) => {
-    const webhookKind = req.headers["pulumi-webhook-kind"];
+    const webhookKind = req.headers["pulumi-webhook-kind"].toString();
```

Comment:
Oh, ok. I was assuming that since these requests originate from the Pulumi service, we'd be able to rely on the presence of this header, but if that's an unsafe assumption, I can add a little error handling here as well.